rockcraft/commands/init.py

```diff
@@ -90,34 +90,59 @@ class InitCommand(AppCommand):
-                # parts:
-                #   flask-framework/dependencies:
-                #     stage-packages:
-                #       # list required packages or slices for your flask application below.
-                #       - libpq-dev
-                #
+                # parts:  # you need to uncomment this line to add or update any part.
+
                 #   flask-framework/install-app:
                 #     prime:
-                #       # by default, only the files in app/, templates/, static/, and app.py
-                #       # are copied into the image. You can modify the list below to override
-                #       # the default list and include or exclude specific files/directories
-                #       # in your project.
+                #       # by default, only the files in app/, templates/, static/, migrate,
+                #       # migrate.sh and app.py are copied into the image. You can modify the list
+                #       # below to override the default list and include or exclude specific
+                #       # files/directories in your project.
                 #       # note: prefix each entry with "flask/app/" followed by the local path.
                 #       - flask/app/.env
                 #       - flask/app/app.py
                 #       - flask/app/webapp
                 #       - flask/app/templates
                 #       - flask/app/static
+                #       # you may need packages to build a python package. Add them here if necessary.
+                #       build-packages:
+                #           # for example, if you need pkg-config and libxmlsec1-dev to build one
+                #           # of your packages:
+                #           - pkg-config
+                #           - libxmlsec1-dev
+
+                # you can add package slices or Debian packages to the image.
+                # package slices are subsets of Debian packages, which result
+                # in smaller and more secure images.
+                # see https://documentation.ubuntu.com/rockcraft/en/latest/explanation/chisel/
+
+                # add this part if you want to add packages slices to your image.
+                # you can find a list of packages slices at https://github.com/canonical/chisel-releases
+                #   flask-framework/runtime-slices:
+                #     plugin: nil
+                #     stage-packages:
+                #     # list the required package slices for your flask application below.
+                #     # for example, for the slice libs of libxml2:
+                #     - libxml2_libs
+
+                # if you want to add a Debian package to your image, add the next part
+                #   flask-framework/runtime-debs:
+                #     plugin: nil
+                #     stage-packages:
+                #     # List required Debian packages for your flask application below.
```

Comment:
```suggestion
                #     # list required Debian packages for your flask application below.
```
was this one upper-cased on purpose?

Reply:
good catch, I did not see it :(